tests/spec_decode/test_utils.py

```diff
@@ -4,10 +4,12 @@
 import torch
 
 from vllm.model_executor.layers.rejection_sampler import RejectionSampler
+from vllm.model_executor.layers.sampler import _get_ranks
```

Comment:
is there a better way we can do this besides importing a private function? maybe copying one of these https://github.com/vllm-project/vllm/blob/main/tests/spec_decode/e2e/test_logprobs.py ?

Reply:
actually this will be faster to test, let's keep it as is